agent/nmap_agent.py

```diff
@@ -394,6 +415,50 @@ def _emit_fingerprints(
                                 data=msg_data,
                             )
 
+    def _connect_to_vpn(self) -> None:
+        """Connect to VPN."""
+        logger.info("Trying to scan the asset with VPN")
+        try:
+            with open(WIREGUARD_CONFIG_FILE_PATH, "w", encoding="utf-8") as conf_file:
+                conf_file.write(cast(str, self._vpn_config))
+            with open(DNS_RESOLV_CONFIG_PATH, "w", encoding="utf-8") as conf_file:
+                conf_file.write(cast(str, self._dns_config))
+
+            self._exec_command(
+                [
+                    "cp",
+                    WIREGUARD_CONFIG_FILE_PATH,
+                    "/etc/wireguard/wg0.conf",
+                ]
+            )
+            self._exec_command(["wg-quick", "up", "wg0"])
+            self._exec_command(["cp", DNS_RESOLV_CONFIG_PATH, "/etc/resolv.conf"])
+            logger.info("connected with %s", WIREGUARD_CONFIG_FILE_PATH)
```

Comment:
I think it would be better to show the country in the logs and not the path `WIREGUARD_CONFIG_FILE_PATH`.